_capabilities/utd-ucla-juice.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Juice"
+excerpt: "Juice is a library for learning tractable probabilistic circuits from data, and using the learned models to build solutions in explainable AI, robustness to missing data, algorithmic fairness, compression, etc. "
+tags:
+  - Analytics
+  - Computer Vision
+  - Explanation Framework
+   
+submission_details:
+  resources: 
+    papers:
+      - title: Juice: A Julia Package for Logic and Probabilistic Circuits
```

Comment:
since yaml uses colons for mapping, you need to place the title in quotations to prevent a syntax error:

```suggestion
      - title: "Juice: A Julia Package for Logic and Probabilistic Circuits"
```